token-lending/program/tests/borrow.rs

```diff
@@ -104,11 +104,18 @@ async fn test_success() {
         .fees
         .calculate_borrow_fees(borrow_amount)
         .unwrap();
-    let sol_fee_balance =
-        get_token_balance(&mut banks_client, sol_reserve.collateral_fees_receiver).await;
+
     assert!(total_fee > 0);
     assert!(host_fee > 0);
-    assert_eq!(total_fee - host_fee, sol_fee_balance);
+
+    let sol_collateral_supply =
+        get_token_balance(&mut banks_client, sol_reserve.collateral_supply).await;
+    assert_eq!(sol_collateral_supply, borrow_amount - total_fee);
```

Comment:
nit: `borrow_amount` is misnamed, it should be `collateral_amount`. Will fix in a follow up